ddp-workspace/projects/ddp-sdk/src/lib/components/user/manageParticipants.component.ts

```diff
@@ -77,7 +79,10 @@ export class ManageParticipantsComponent implements OnDestroy {
     }
```

Comment:
github not letting me comment on it, but line 68:
can be changed to:
```typescript
mergeMap(x => this.serviceAgent.getList()),
```
Should fix deprecated warning I am seeing.
(I know this is not one of your changes, but since you are in there already...)